backend/hyperglosae/src/updates/document.js

```diff
@@ -0,0 +1,22 @@
+function(doc, req) {
+    const body = JSON.parse(req.body);
+    if (!doc) {
+        doc = Object.assign({
+            _id: req.id,
+            _rev: req.rev,
+            history: [{
+                user: req.userCtx.name,
+                date: new Date().toISOString(),
+            }]
+        }, body);
+    } else {
+        const history = (doc.history && doc.history[0]) || {
+            user: 'Unknown',
+            date: 'Unknown',
+        };
+        doc = Object.assign({}, body, {
+            history: [history]
+        });
```

Comment:
Idem. If CouchDB interpreter allows it, you can express this as (please test it first and adapt):
```
doc = {
  ...body,
  history: doc.history || {user: 'Unknown', date:'Unknown'}
}
```
By the way, I wonder if this wouldn't be better to have an empty history (`[]`) instead of special values like `Unknown` (esp. when the interface is to be internationalized).
By the way (too), this case would probably deserve a test scenario...